### DISCUSSION TITLE
Must Use Inpainting? | ControlNet Usable?

### DISCUSSION THREAD

**Discussion by JimmyTime** [volotat/SD-CN-Animation#41] [Q&A]
I saw mention in the writeup that the examples used 1.5 inpainting. Is it recommended using an inpainting model only?

Also, I seem to be getting soft errors on every frame telling me about some of the typical extensions I have (currently) turned off in the img2img and txt2img tabs (such as aesthetic gradients, composable lora, and dynamic thresholding). Are these able to be used with this extension? If so how? Img2Img tab?

**Comment by volotat** [comment]
There are several places where inpainting is used in the code, and theoretically an inpainting model might work better, but from my testing I didn't notice it. I usually use the 'realisticVisionV13_v13' model and it works well enough for me. Regarding the extensions I haven't tested it myself, but they should work exactly like in img2img cases. Try to use lora tokens in prompt and see if it works.
